connect/templates/camunda7/src/main/java/io/miragon/miranum/connect/elementtemplate/c7/Camunda7ElementTemplateGenerator.java

```diff
@@ -31,38 +32,51 @@ public ElementTemplateGenerationResult generate(ElementTemplateInfo elementTempl
 
         // Add properties for input parameters
         for (var inputProperty : elementTemplateInfo.getInputProperties()) {
-            var property = createPropertyWithValue(inputProperty, false);
+            var property = createInputParameterProp(inputProperty);
             elementTemplate.getProperties().add(property);
         }
 
         // Add properties for output parameters
         for (var outputProperties : elementTemplateInfo.getOutputProperties()) {
-            var property = createPropertyWithValue(outputProperties, true);
+            var property = createOutputParameterProp(outputProperties);
             elementTemplate.getProperties().add(property);
         }
 
         var json = CamundaC7ElementTemplateConverter.toJsonString(elementTemplate);
-        return new ElementTemplateGenerationResult(elementTemplateInfo.getId(), elementTemplateInfo.getVersion(), json, TargetPlatform.camunda7);
+        return new ElementTemplateGenerationResult(elementTemplateInfo.getId(), elementTemplateInfo.getVersion(), json);
     }
 
-    private Property createPropertyWithValue(ElementTemplatePropertyInfo info, boolean output) {
-        var value = output ? info.getLabel() + "Result" : "";
-        var bindingType = output ? Binding.Type.CAMUNDA_OUTPUT_PARAMETER : Binding.Type.CAMUNDA_INPUT_PARAMETER;
+    private Property createInputParameterProp(ElementTemplatePropertyInfo info) {
         var property = new Property()
-                .withLabel(info.getLabel())
+                .withLabel("Input: %s".formatted(info.getLabel()))
```

Comment:
Das Input: brauchts nicht. Sonst steht das auch überall in der Engine vor den Parametern. Evtl könnte man hier aus

camelCaseValue -> Camel Case Value 

machen. So mach ichs gerade manuell.

Reply:
Das Input steht nur bei dem Label und das sieht man in der .bpmn Datei nicht.

| Modeler | BPMN |
|-------------|----------|
| ![image](https://github.com/Miragon/miranum/assets/100686712/42a88814-bf9b-4805-a875-10c9780e2836) | ![image](https://github.com/Miragon/miranum/assets/100686712/5c15da89-a17b-4022-a503-747ec1be0549) |

Oder was meinst du, dass das in der Engine steht?

Reply:
ok, never mind. Ist gut so :)